1.8/administration/access-node/tunnel.md

```diff
@@ -1,50 +1,113 @@
 
 # VPN
-dcos tunnel provides you with access to the DNS, masters, and agents from within the cluster. OpenVPN requires root privileges in order to configure these routes.
+dcos tunnel provides you with full access to the DNS, masters, and agents from within the cluster. OpenVPN requires root privileges to configure these routes.
 
 # Using dcos tunnel
 
 ## Prerequisites
 * DC/OS 1.8+
 * Docker
-* Only Linux and OS X support currently
-* Requires [SSH access](/1.8/administration/access-node/sshcluster/) (key authentication only)
-* Fpr VPN you must have [the OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) installed.
-* The dcos tunnel package must be installed from Universe: `dcos package install tunnel-cli --cli`
-
-
-*** For commands that port forward, append `.mydcos.directory` to the end of your domain
-e.g. http://example.com:8080/?query=hello becomes http://example.com.mydcos.directory:8080/?query=hello
-SOCKS and HTTP port forward, VPN does not
-The reason for this (does not need to be documented) is that the tunnel sends traffic that goes to `localhost` through SSH. But a normal url does not send traffic to `localhost`, it sends traffic to wherever you asked it to go (e.g. marathon.mesos). `mydcos.directory` is a special domain that always resolves to `localhost`, and within DC/OS we cut this part of the URL off before resolving it.
-
-SOCKS
-`dcos tunnel socks`
-Port forwards! See above.
-Configure application to use proxy on port 1080
-
-HTTP
-`sudo dcos tunnel http`
-Port forwards! See above.
-It will just work
-Otherwise, if configured to not use port 80, it does not require root access (so just `dcos tunnel…`) and it will require applications to be configured to use the proxy.
-
-OpenVPN
-`sudo dcos tunnel vpn`
-It should just work
-There is an option enabled that auto-configures DNS, but it doesn’t work on OSX so DNS has to be configured manually with the printed ip addresses.
+* Requires [SSH access](/1.8/administration/access-node/sshcluster/) (key authentication only).
+* For VPN functionality, you must have [the OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) installed.
+* The dcos tunnel package must be installed: run `dcos package install tunnel-cli --cli` from the DC/OS CLI.
+
+**Note:** * Only Linux and OS X are supported currently.
+
+##  SOCKS
+Run the following command from the DC/OS CLI:
+
+```
+$ dcos tunnel socks
+```
+
+Configure your application to use the proxy on port 1080: `127.0.0.1: 1080`
+
+### Port Forwarding
+The SOCKS proxy works by port forwarding. Append `.mydcos.directory` to the end of your domain when you enter commands. For instance, `http://example.com:8080/?query=hello` becomes `http://example.com.mydcos.directory:8080/?query=hello`.
```

Comment:
Oh, I realize now that this paragraph was specific to SOCKS, I think port forwarding can be a separate thing that gets reused between HTTP and SOCKS.